### DISCUSSION TITLE
What is the impact of this vuln

### DISCUSSION THREAD

**Discussion by SAGEof6iixPATHS** [projectdiscovery/nuclei-templates#1477] [Q&A] [answered]
Hello!

What is the impact of this vulnerability?
misconfiguration/airflow-api-exposure.yaml

Any reports or write-ups?

**Comment by ehsandeep** [comment, marked as answer]
https://nvd.nist.gov/vuln/detail/CVE-2020-13927

**Reply by toufik-airane** [reply]
The template _misconfiguration/airflow-api-exposure.yaml_ becomes [CVE-2020-13927](https://github.com/projectdiscovery/nuclei-templates/blob/master/cves/2020/CVE-2020-13927.yaml).

According to NIST, It's a critical issue.
https://nvd.nist.gov/vuln/detail/CVE-2020-13927

Exploit: https://packetstormsecurity.com/files/162908/Apache-Airflow-1.10.10-Remote-Code-Execution.html

**Reply by ehsandeep** [reply]
The linked exploit is for CVE-2020-11978, and we have a [template](https://raw.githubusercontent.com/projectdiscovery/nuclei-templates/master/cves/2020/CVE-2020-11978.yaml) for that as well.
